### DISCUSSION TITLE
Unresolved reference 'CSSTag'

### DISCUSSION THREAD

**Discussion by Stepashka20** [AllenDang/giu#644] [Q&A] [answered]
Hi! I took the code from the example https://github.com/AllenDang/giu/tree/master/examples/CSS-styling , but the IDE shows **"undefined: gia.CSSTag"**. And also when compiling:
```
.\main.go:14:7: undefined: giu.CSSTag
.\main.go:17:7: undefined: giu.CSSTag
.\main.go:25:16: undefined: giu.ParseCSSStyleSheet
```

main.go
```golang
package main

import (
	_ "embed"

	"github.com/AllenDang/giu"
)

//go:embed style.css
var cssStyle []byte

func loop() {
	giu.Window("Window").Layout(
		giu.CSSTag("button").To(
			giu.Button("HI! I'm a button styled with CSS"),
		),
		giu.CSSTag("label").To(
			giu.Label("I'ma  normal label"),
		),
	)
}

func main() {
	wnd := giu.NewMasterWindow("CSS Style [example]", 640, 480, 0)
	if err := giu.ParseCSSStyleSheet(cssStyle); err != nil {
		panic(err)
	}
	wnd.Run(loop)
}
```
![image](https://user-images.githubusercontent.com/40739871/216833315-432dc2c3-4eb4-42a0-88a0-bfed2ab6c586.png)
![image](https://user-images.githubusercontent.com/40739871/216833318-587752ab-e15e-4290-b7de-7e33cdf4c808.png)

What am I doing wrong?

**Comment by gucio321** [comment, marked as answer]
@Stepashka20 most likely you are using too old giu version (go get github.com/AllenDang/giu@master)

**Reply by Stepashka20** [reply]
Oh, thank you so much
